src/math/exp_fcn_hifi.c

```diff
@@ -302,10 +367,10 @@ int32_t exp_fixed(int32_t x)
 	int i;
 	int n = 0;
 
-	if (x < Q_CONVERT_FLOAT(-11.5, 27))
+	if (x < exp_hifi_q_convert_float(-11.5, 27))
```

Comment:
this is not build with pre-compiler stage? could you show the asm code to compare?

Reply:
Sorry, I don't get the necessity in this situation; could you please explain?

Reply:
I mean why not directly use:Q_CONVERT_FLOAT(-11.5, 27)?